Detector/DetSensitive/tests/options/testDd4hepFullParticleAbsSD.py

```diff
@@ -0,0 +1,63 @@
+from Gaudi.Configuration import *
```

Comment:
It's super you create the test for the new SD class!
The only comment - to make it an actual test - you need to add it also to Detector/DetSensitive/CMakeLists.txt